testing/trino-product-tests-launcher/src/main/resources/docker/presto-product-tests/conf/environment/singlenode-delta-lake-databricks91/delta.properties

```diff
@@ -0,0 +1,8 @@
+connector.name=delta_lake
```

Comment:
`singlenode-delta-lake-databricks-snappy` (or `singlenode-delta-lake-databricks-pre-11`)  would allow us to have one 
copy of it (one new directory), and also would indicate what the difference is